server/src/repositories/media.repository.ts

```diff
@@ -54,22 +59,63 @@ export class MediaRepository implements IMediaRepository {
         return false;
       }
     }
-
     return true;
   }
 
+  async cloneExif(input: string, output: string): Promise<boolean> {
```

Comment:
Copying *all* of the metadata seems unnecessary. It adds to the size of the output, and there are surely metadata entries that are correct for the RAW, but wrong for the extracted image.

Can you narrow down which field(s) specifically need to be added? I'd like to see if we can avoid that exiftool.read call and just apply the data in exifInfo that we already have from metadata extraction.

Reply:
I still don't see what the point is to including all the EXIF fields in the image. This is a generated image with a specific purpose. It doesn't need to have anything beyond what's need to display it correctly. If anything, the fact that the extracted image has metadata is a downside that prevents sharing due to metadata leakage.

Reply:
This fullsize image extracted from RAW can be more than just specifically for preview. I'm thinking of adding features like "download as JPG" and "remove RAW but keep JPG to cleanup space" for RAW files later, which would be very handy for workflows, and it's important to preserve full EXIF fields.

As for EXIF leaking, this is indeed an issue when a shared link disallows downloading and hides EXIF but original JPG files can be acquired via `thumbnail?size=fullsize`, let me address this issue.

But anyway, this `cloneExif()` is already removed for now.

Reply:
I see what you're going for, but I think this should be handled when those features are actually implemented. It can be done as part of the request to download them, etc.

Reply:
Good point, let me keep only colorspace and orientation EXIF in this PR.

Reply:
Also I have added ` Permission.ASSET_DOWNLOAD` requirement for `thumbnail?size=fullsize` to prevent leaking EXIF via original files